### PR TITLE
correct command to cd to jupyter conf dir

### PR DESCRIPTION
**Pull request by gmanchon**
The env var `$JUPYTER_CONFIG_DIR` allows to define a specific conf dir for jupyter but does not seem to contain the path to the jupyter conf dir

Addresses https://github.com/lewagon/data-setup/issues/105

### CODE REVIEW

File: WINDOWS.md [hunk 1]
```diff
@@ -471,7 +471,7 @@ Then, let's disable warning for copy-pasting commands between Windows and Ubuntu
 
 :warning: Do not forget the comma at the end of the line!
 
-You can save these changes by pressing `CTRL` + `V`
+You can save these changes by pressing `CTRL` + `S`
```
Comment (krokrob) [inline WINDOWS.md:474]: Does that comes from a `lewagon/setup` partial?

**Review by krokrob — COMMENTED** [review]
lgtm

File: WINDOWS.md (hunk 1 above)
Comment (gmanchon) [reply WINDOWS.md:474]: yes, looks like a github action 👌

Comment (gmanchon) [reply WINDOWS.md:474]: yes, looks like this was injected by a github action from the web partial `setup/windows_terminal`

**Comment by gmanchon** [comment]
@krokrob I added the missing mkdir

**Pull request merged**